packages/yoastseo/src/languageProcessing/researches/findKeywordInFirstParagraph.js

```diff
@@ -33,6 +38,7 @@ export default function( paper, researcher ) {
 		foundInOneSentence: false,
 		foundInParagraph: false,
 		keyphraseOrSynonym: "",
+		introduction: firstParagraph,
```

Comment:
Don't think this is necessary to return, right? 😃 I don't see it used yet 🤔

Reply:
This will be used in Premium. So instead of running `getParagraphs`, in premium we can run `getKeywordInFirstParagraph` research to get the `introduction`. I find that this is a more reliable approach to get a matched introduction block since at the moment we cannot pass a list of nodes that needs to be filtered out when running `getParagraphs` research via `runResearch`